monitoring/ecs_dashboard/Makefile

```diff
@@ -16,15 +16,14 @@ $(val $(call stack_setup))
 
 ECS_DASHBOARD = $(ROOT)/monitoring/ecs_dashboard
 
-$(ROOT)/.docker/ecs_dashboard:
-	$(ROOT)/builds/build_ci_docker_image.py \
-		--project=ecs_dashboard \
-		--dir=$(ECS_DASHBOARD)/client \
-		--file=$(ECS_DASHBOARD)/client/Dockerfile
+build_deployment_container:
+	docker build \
+		-f client/Dockerfile client/ \
+		--tag ecs_dashboard
 
-ecs_dashboard_client-publish: $(ROOT)/.docker/ecs_dashboard
```

Comment:
We use this dotfile as the target so we only have to build it once. After the target completes, you run `mkdir -p $(ROOT)/.docker; touch $(ROOT)/.docker/ecs_dashboard`.

It's not massively important because we don't run this very often, but might be worth changing.